### ISSUE TITLE
Make a 04-resources-for-learning.md

### ISSUE THREAD

**Issue by iamciera** [Reproducible-Science-Curriculum/rr-version-control#16]
This would be something they could read on their own.  It should include the simplest way to further learning git. Only the best tutorials. 

It might also be nice to include the pros and cons between using git through 1. GUI 2. Command line 3. Rstudio.

**Comment by iamciera** [comment]
This is basically a note that might help when teaching Git.  

I am currently learning Docker and find it really useful to have a terminology list open while listening to the lectures. I think all the new terminology makes learning Git, Docker, ect. a real stuggle.  Maybe we can create a list or link to this [terminology list](https://help.github.com/articles/github-glossary/) at the top of the tutorials.

**Comment by mine-cetinkaya-rundel** [comment]
Good idea! I agree that the terminology can be a frustration point, and this would help.

**Comment by lazappi** [comment]
Should we collect some resources here? Here are a few I found after a quick look, some of them could overlap so might not need them all in the final list:

* git
  - [tryGit](https://try.github.io/levels/1/challenges/1) - Interactive git tutorial
  - [git - the simple guide](https://rogerdudler.github.io/git-guide/) - quick, colourful guide to basic git
  - [Visual git reference](https://marklodato.github.io/visual-git-guide/index-en.html) - Illustrations of what git commands do
  - [Think like (a) git](http://think-like-a-git.net/) - More humorous guide to git, includes some graph theory
  - [Pro Git book](https://git-scm.com/book/en/v2/) - All the technical details of git
  - [stackoverflow - git](https://stackoverflow.com/questions/tagged/git) - git on stackoverflow
* Github
  - [Github guides](https://guides.github.com/) - Guides to all the things you can do on Github
  - [Github help](https://help.github.com/) - Github help pages
  - [stackoverflow - Github](https://stackoverflow.com/questions/tagged/github) - Github on stackoverflow
* Terminology
  - [Github glossary](https://help.github.com/articles/github-glossary/) - list of git and Github terms
* Cheatsheets from:
  - [Github](https://services.github.com/on-demand/downloads/github-git-cheat-sheet.pdf)
  - [Tower](https://www.git-tower.com/blog/git-cheat-sheet/)
  - [Zero Turnaround](https://zeroturnaround.com/rebellabs/git-commands-and-best-practices-cheat-sheet/)
  - [Simple guide](https://rogerdudler.github.io/git-guide/files/git_cheat_sheet.pdf)
